modules/components/src/Aggs/AggsWrapper.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Component from 'react-component-component';
+import './AggregationCard.css';
+
+export default ({ children, collapsible = true, displayName }) => (
+  <Component initialState={{ isCollapsed: false, showingMore: false }}>
+    {({ setState, state: { isCollapsed, showingMore } }) => (
+      <div className="test-term-aggregation aggregation-card">
+        <div
+          className={`title-wrapper ${isCollapsed && 'collapsed'}`}
```

Comment:
make this `className={`title-wrapper ${isCollapsed ? 'collapsed' : ''}`}` or you will end up with a `false` class :)